examples/notebooks_new_field/introduction.ipynb

```diff
@@ -0,0 +1,1099 @@
+    "import matplotlib.ticker       as mticker    \n",
+    "from   mpl_toolkits.axes_grid1 import make_axes_locatable\n",
+    "from   matplotlib              import cm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now import all the Devito package to have more freedom of work."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from devito import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From Devito's library of examples, we import some specific functions, such as:\n",
+    "\n",
+    "- *TimeAxis* which generates a temporal structure;\n",
+    "- *RickerSource* for the setup of the source term;\n",
+    "- *Receiver* which generates a structure of receivers;\n",
+    "\n",
+    "So, we import these sample libraries with the following commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from   examples.seismic        import TimeAxis\n",
+    "from   examples.seismic        import RickerSource\n",
+    "from   examples.seismic        import Receiver"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We define the spatial parameters to be used in the simulations. In Devito, the mesh data must be defined taking into account *meter* as the unit of measurement. Thus, we have the following variables:\n",
+    "\n",
+    "- *nptx* and *nptz* correspond to the number of points (nx+1 and nz+1) in the $x$ and $z$ directions, respectively;\n",
+    "- *x0* and *x1* determine the starting and ending point in the direction $x$;\n",
+    "- *z0* and *z1* determine the starting and ending points in the $z$ direction;\n",
+    "- *compx* and *compz* the lengths of the domain in the directions $x$ and $z$, respectively. These values are obtained from *x0*, *x1*, *z0* and *z1*;\n",
+    "- *hx* and *hz* correspond to $\\Delta x$ and $\\Delta z$, respectively. These values are calculated in terms of the start/end points and the number of points in each direction, respectively;\n",
+    "\n",
+    "These variables are defined in the code as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nptx   = 861\n",
+    "nptz   = 311\n",
+    "x0     = 13650.\n",
+    "x1     = 56650. \n",
+    "compx  = x1-x0\n",
+    "z0     = 0.\n",
+    "z1     = 9920.\n",
+    "compz  = z1-z0;\n",
+    "hx     = (x1-x0)/(nptx-1)\n",
+    "hz     = (z1-z0)/(nptz-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the spatial parameters are defined, we start the construction of the *spatial grid* using the Devito structure. The required parameters are:\n",
+    "\n",
+    "- *origin* is the point of origin of the grid;\n",
```

Comment:
A vector as well, I think you can safely use the defs from the docstrings as well.

Reply:
The objective is to detail all the parameters that a new user can be change and construct a new example and/or test using these notebooks.